lightningd/invoice.c

```diff
@@ -39,7 +39,8 @@ static void json_add_invoice(struct json_stream *response,
 			     const struct invoice_details *inv)
 {
 	json_add_escaped_string(response, "label", inv->label);
-	json_add_invstring(response, inv->invstring);
+	if (inv->invstring)
+		json_add_invstring(response, inv->invstring);
```

Comment:
```suggestion
	if (inv->invstring) {
	        log_unusual("invstring null this should be cause from a very old invoice (back in 2018 maybe?)")
		json_add_invstring(response, inv->invstring);
         }
```

Not sure that the `log_unusual` syntax is correct, but you got may point